### ISSUE TITLE
Feature request: Seed cache-image

### ISSUE THREAD

**Issue by Froelund** [kfiroo/react-native-cached-image#22]
Hi!

In our app, we have the following scenario:

1. Upload an image, from the phone using `react-native-fetch-blob`. Getting an url back as a reference.
2. Send the reference url, to other devices.
3. Showing the image on all devices, including the sending device.

So - As an optimization for the sending device, we would like to be able to supply the local image from the first step, as a cache for the url returned.

```
var supplyResponse;
try{
    supplyResponse = ImageCacheProvider.seedCache('https://remote-image-service.com/img-1.png', '/some-local-path/img-1.png');
} catch(err) {
    //Could not read local file
}

```

`ImageCacheProvider.seedCache` would then copy the file `/some-local-path/img-1.png` to the appropriate path.

**Comment by kfiroo** [comment]
@Froelund Sounds good! Want to create a PR for that?

**Comment by Froelund** [comment]
Sure - As soon as I find the time :)
Just wanted to know if you liked the idea.

**Comment by hhravn** [comment]
#27

**Comment by kfiroo** [comment]
Released with v1.3.2
